x-pack/metricbeat/module/panw/routing/_meta/fields.yml

```diff
@@ -0,0 +1,150 @@
+- name: routing
+  type: group
+  release: beta
+  description: >
+    routing
+  fields:
+    - name: bgp.peer_name
+      type: keyword
+      description: >
+        The name of the current peer in the BGP peer group
+    - name: bgp.virtual_router
+      type: keyword
+      description: >
+        The virtual router with which the BGP peer is associated
+    - name: bgp.peer_group
+      type: keyword
+      description: >
+        The name of the BGP peer group this peer belongs to
+    - name: bgp.peer_router_id
+      type: ip
+      description: >
+        BGP peer router ID
+    - name: bgp.remote_as_asn
+      type: long
+      description: >
+        The remote Autonomous System (AS) number of the peer
+    - name: bgp.status
+      type: keyword
+      description: >
+        The BGP session status, e.g., "Established" means the session is up and running
+    - name: bgp.status_duration
```

Comment:
```suggestion
    - name: bgp.status_duration.sec
```

Reply:
So I guess I don't understand how this is supposed to work. 
I defined in fields.yml:
```
name: bgp.status_duration.sec
      type: long
```

The set it in go like this:
`
"bgp.status_duration.sec":        entry.StatusDuration,`

But get the error:
```
failed to parse field [panw.routing.bgp.status_duration] of type [long] in document with id 'a6WZ55EBLzEFnbLBRRn-'. 
    Preview of field's value: '{sec=0}'\",\"caused_by\":{\"type\":\"illegal_argument_exception\",\"reason\":\
    "Cannot parse object as number\"}}, dropping event!",
```

Reply:
OK, I see - don't include the units prefix when setting the field in go.

Reply:
This happens because there is no explicit mapping, this should be fixed by the integration where we add mappings for all these fields. The suffix should be set when building the documents as well, not only in `fields.yml`. Maybe we can keep them like they are for now and come back to this in the future. @agithomas WDYT?